explainer.md

```diff
@@ -72,9 +89,15 @@ These signatures can be used in three related ways:
 3. When signed by a raw public key, the signatures enable [signature-based
    subresource integrity](https://github.com/mikewest/signature-based-sri).
 
-We publish [periodic snapshots of this
-draft](https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl)
-so that test implementations can interoperate.
+Signed exchanges can also be sent to a client in three ways:
+
+1. Using a `Signature` header in a normal HTTP response. This way is used for
+   non-origin signatures and to provide an origin-trusted signature to
+   intermediates.
+1. Enveloped into the `application/signed-exchange` content type. In this case,
+   the signed exchange has both the logical URL of its embedded request, and the
+   physical URL of the envelope itself.
+1. In an HTTP/2-Pushed exchange.
```

Comment:
Until their interaction with Push is specified, I wonder if it's better to just leave this out.

Reply:
In the explainer, I think it makes sense to lay out the goal, even though we haven't written down exactly how we're getting there yet. In the loading specification, I think you're right that I should only add the PUSH route when I can actually write down how the browser processes it.

Does that make sense? If you're not convinced, I'm not dead set on keeping this entry.

There's also a question of whether it makes sense to target Push in any new feature...